docs/guides/culture.html.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Cultural Changes
+subtitle: Plugin creation
+layout: guide
+order: 0
+---
+
+### Introducing Danger
+
+It can be easy to try and jump straight from no Dangerfile to a 200 line complex set of cultural rules. We'd advise against first coming up with a really long checklist of rules for Danger, and then trying to introduce them all at-once. The entire team may have agreed on the changes, but slower gradual integration has worked better for teams new to working with Danger.
+
+At Artsy we've found that first just integrating Danger with a single simple rule (like checking for a CHANGELOG entry) then starting to introduce them piece-meal with different contributors has made it easier to go from "Ah, we shouldn't do that again" to "Oh, we could make a Danger rule for that" to "Here's the PR". 
```

Comment:
Yeah, it's to encourage the idea that people should try to avoid becoming the sole person on the team who makes changes to Danger rules - they're you team rules, your team should feel free to update them.